### DISCUSSION TITLE
[@types/styled-components-react-native] ships outdated @types/react-native

### DISCUSSION THREAD

**Discussion by levino** [DefinitelyTyped/DefinitelyTyped#57729] [Issues with a @types package]
When trying to use @types/styled-components-react-native with @types/react-native, I get type mismatches. I have @types/react-native@0.66.9 installed and @types/styled-components-react-native@5.1.3

[a comment by an automated account is not shown]

**Comment by rob0rt** [comment]
I just ran into this myself and was able to get around it by forcing the correct version with a resolution block in my package.json:
```
"resolutions": {
    "@types/react": "^17",
    "@types/react-native": "^0.66.15"
},
```
https://classic.yarnpkg.com/en/docs/selective-version-resolutions/
